validator/validator-main.protoascii

```diff
@@ -4359,6 +4359,8 @@ tags: {
   }
   attrs: {
     name: "type"
+    # When updating this blacklist be sure to update the sanitizer BLACKLISTED_TAG_SPECIFIC_ATTR_VALUES
+    # as these values also need to be stripped when used in templates.
```

Comment:
please add a link to the file that BLACKLISTED_TAG_SPECIFIC_ATTR_VALUES appears in

Reply:
done. Thanks